### DISCUSSION TITLE
Error when I try Integrating FRET with Copilot

### DISCUSSION THREAD

**Discussion by lht011129** [nasa/ogma#167] [Q&A]
![ogma-demo](https://github.com/user-attachments/assets/c08ed79a-6887-4845-982e-881f45e7c5fd)
As I tried integrating FRET with Copilot as gif I uploaded, I missed the error called "error: Variable not in scope: notPreviousNot :: Stream Bool -> Stream Bool". I have no idea about this error and  could you please provide some instructions or complete example? Thank you very much!

**Comment by ivanperez-keera** [comment]
This should now be solved in the version in the repo. Please be aware that the new interface to compile these files is:

```
$ ogma standalone --file-name some_file.json
```
